common/importer/chrome_importer_utils_linux.cc

```diff
@@ -21,6 +21,28 @@ base::FilePath GetChromeUserDataFolder() {
   return result;
 }
 
+base::FilePath GetChromeBetaUserDataFolder() {
+  base::FilePath result;
+  if (!base::PathService::Get(base::DIR_HOME, &result))
+    return base::FilePath();
+
+  result = result.Append(".config");
+  result = result.Append("google-chrome-beta");
```

Comment:
I don't think it matter to be honest, but this could be just:

```
result = result.Append(".config").Append("google-chrome-beta");
```

Maybe you did the other way to be explicit, which is fine as well.